packages/widget-v2/src/pages/SwapPage/SwapPage.tsx

```diff
@@ -23,19 +32,28 @@ export const SwapPage = () => {
   const [{ data: assets }] = useAtom(skipAssetsAtom);
   const [{ data: chains }] = useAtom(skipChainsAtom);
   const [destinationAsset, setDestinationAsset] = useAtom(destinationAssetAtom);
-
+  const [_, setSwapDirection] = useAtom(swapDirectionAtom);
```

Comment:
I'm just learning this myself, we should use [useSetAtom](https://jotai.org/docs/core/use-atom#usesetatom)if we dont need to actually get the value